src/test/steps/happy-path.ts

```diff
@@ -1,63 +1,44 @@
 import { State } from '../../main/app/case/definition';
-import {
-  APPLICANT_2,
-  APPLY_FINANCIAL_ORDER,
-  HAS_RELATIONSHIP_BROKEN_URL,
-  HOW_DO_YOU_WANT_TO_RESPOND,
-  RESPONDENT,
-} from '../../main/steps/urls';
-
-import {
-  checkOptionFor,
-  iAmOnPage,
-  iClearTheForm,
-  iClick,
-  iSetApp2UsersCaseTo,
-  iSetRespondentUsersCaseTo,
-  iSetTheUsersCaseTo,
-} from './common';
+import { ADDRESS_PRIVATE, HOME_URL } from '../../main/steps/urls';
+import { autoLogin, config as testConfig } from '../config';
+
+import { checkOptionFor, iAmOnPage, iClearTheForm, iClick, iSetTheUsersCaseTo } from './common';
 import { iEnterTheUkAddress } from './postcode';
 
 const { I } = inject();
 
 Given("I've already completed the form using the fixture {string}", async (fixture: string) => {
+  I.amOnPage(HOME_URL);
   const fixtureJson = require(`../functional/fixtures/${fixture}`)[fixture];
 
   await iSetTheUsersCaseTo(fixtureJson);
 
-  const url = await I.grabCurrentUrl();
-  I.amOnPage(APPLY_FINANCIAL_ORDER);
-  iClick('Continue');
-  I.amOnPage(url);
+  I.amOnPage(ADDRESS_PRIVATE);
+  I.click('I do not need my contact details kept private');
```

Comment:
is there a reason for this?

i feel like the data should already exist and clicking continue should be enough.

Reply:
same for below

Reply:
@velimd yes it's because when we update the case data using the iSetTheUsersCaseTo we need to then go to a page and click continue on it so that we fetch the data from ccd on the 'Continue'. Previously we were logging the user out and then back in but I thought it would save time this way. 

The reason I chose the ADDRESS_PRIVATE page to do it was because it's one of the few pages accessible for app1, app2 and the respondent